includes/omise-wc-setting.php

```diff
@@ -0,0 +1,100 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+return array(
+	'enabled' => array(
+		'title'       => __( 'Enable/Disable', $this->gateway_name ),
+		'type'        => 'checkbox',
+		'label'       => __( 'Enable Omise Payment Module.', $this->gateway_name ),
+		'default'     => 'no'
+	),
+	'sandbox' => array(
+		'title'       => __( 'Sandbox', $this->gateway_name ),
```

Comment:
@guzzilar Finally, all of `$this` reference in this file will be removed and it will be changed to use the common function.

Like this, https://github.com/omise/omise-woocommerce/blob/feature/i18n/omise-wc-gateway.php#L47.